src/main/java/Deadline.java

```diff
@@ -0,0 +1,25 @@
+public class Deadline extends Task {
+    /**
+     * Constructs a Deadline task with the specified description and deadline
+     *
+     * @param description The description of the task provided by the user
+     * @param deadline The deadline of the task provided by the user
+     */
+    private String deadline;
+    public Deadline(String description, String deadline) {
+        super(description);
+        this.deadline = deadline;
+    }
+
+    /**
+     * Returns a string representation of the Deadline object.
+     * The string includes a "[D]" prefix to indicate that this is a Deadline task,
```

Comment:
Good that you have included a javadoc for the modified toString() method even though it is not required to. According to coding standards maybe you could consider to remove the @return line and only provide additional details about your modification